### ISSUE TITLE
anyone can run pyinstaller 3.1.1, 3.0 or 2.1 successfully in AIX?

### ISSUE THREAD

**Issue by redbearder** [pyinstaller/pyinstaller#1927]
I have try 3.1.1, 3.0, 2.1
only 2.1 can be install on AIX 6.1 successfully, but I have to change little code and rename  libpython2.7.so to (libpython2.7.so). although I can run this execution file on compilation machine. But I can not distribute package, because I found no dynamic lib files like _md5.so and so on.
Then I try 3.1.1 and 3.0, even latest dev version. all can not be compile on bootloader phase
errors is:

```
../../src/pyi_pythonlib.c: In function 'pyi_pylib_load':
../../src/pyi_pythonlib.c:70:10: error: 'p' undeclared (first use in this function)
../../src/pyi_pythonlib.c:70:10: note: each undeclared identifier is reported only once for each function it appears in
```

try to fix it but failed

anyone can help me and tell me where is the rootcause 
how to fix it
appreciated

**Comment by htgoebel** [comment]
@goodtiding5 : Can you please have a look at this? There seams to be a variable definition missing in `pyi_pythonlib`, see [your change on this](https://github.com/pyinstaller/pyinstaller/commit/2d072a2581988ee0a1f41f05184ca3597ed76863#diff-72db097bb39aa4ba1dde09f1fbadcace).

**Comment by htgoebel** [comment]
New news in this for more then a year. closing. If you feel this is still an issue with the current release, please open a brand new issue.
